src/solution.c

```diff
@@ -429,6 +434,56 @@ static void sol_thd_sleep(systime_t *deadline, systime_t interval)
   chSysUnlock();
 }
 
+/** Calculates Iono and Tropo correction
+ * \param n_ready_tdcp The number of measurements written to `m_tdcp`
+ * \param nav_meas_tdcp Pointer to measurements array
+ * \param pos_ecef position in WGS84 Earth Centered, Earth Fixed (ECEF) Cartesian
+ * \param pos_llh position in WGS84 geodetic coordinates*/
+void calc_iono_tropo(u8 n_ready_tdcp,
```

Comment:
Please, find PR for public libswiftnav here https://github.com/swift-nav/libswiftnav/pull/362